modules/atomic-widgets/props-resolver/transformers/settings/link-transformer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropsResolver\Transformers\Settings;
+
+use Elementor\Modules\AtomicWidgets\PropsResolver\Transformer_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Link_Transformer extends Transformer_Base {
+	public function transform( $value, $key ): array {
+		if ( empty( $value['enabled'] ) ) {
+			return [];
+		}
+
+		if ( empty( $value['href'] ) ) {
+			throw new \Exception( 'Url is not provided.' );
+		}
+
+		$link_attrs = [
+			'href' => esc_url( $value['href'] ) ?? '',
```

Comment:
```suggestion
			'href' => esc_url( $value['href'] ),
```
`esc_url` will return an empty string anyway.